app/models/in_person_enrollment.rb

```diff
@@ -34,7 +34,7 @@ def needs_usps_status_check?(check_interval)
 
   # Returns the value to use for the USPS enrollment ID
   def usps_unique_id
-    user_id.to_s
+    user.uuid.delete('-').slice(0, 18)
```

Comment:
This is just a temporary solution, there's a separate ticket to think through the best way to generate unique IDs